urlaf656038a0_Salihan04_MOBLIMA_tgz-pJ8-com_moblima_util_dataTestJ8/src/com/moblima/presentation/SystemSetting_CLI.java

```diff
@@ -0,0 +1,244 @@
+		int yr,mth,day,hr,min;
+		int id;
+		Calendar calendar = Calendar.getInstance();
+		
+		System.out.println("");
+		System.out.println("=================================");
+		System.out.println("Public Holiday Settings");
+		System.out.println("=================================");
+		System.out.println("");
+		System.out.println("Current list of public holidays: ");
+		System.out.println("");
+		displayListOfPublicHolidays();
+		System.out.println("");
+		System.out.println("=============================="); 
+		System.out.println("Please State what you would like to: ");
+		System.out.println("1: Create New Public Holiday");
+		System.out.println("2: Remove a Public Holiday");
+		System.out.println("3: Back");
+		System.out.println("=====================================");
+		System.out.print("Please enter your choice : ");
+		settingsToChangeChoice = ConsoleReader.readIntInput();
+		switch(settingsToChangeChoice){
+			case 1:
+				SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
+				System.out.print("Please enter the date (DD/MM/YYYY) : ");
+				date = ConsoleReader.readDateInput();
+				
+				System.out.print("Please enter The Name Of the Public Holiday : ");
+				name = ConsoleReader.readString();
+				publicHolidayBL.setPublicHoliday(date, name);
+				break;
+			case 2:
+				System.out.print("Please enter the public holiday id to remove : ");
+				id = ConsoleReader.readIntInput();
+				publicHolidayBL.removePublicHoliday(id);
+				break;
+			case 3:
+				System.out.println("Exiting....");
+				break;
+			default:
+				System.out.println("invalid choice");
+			
+		}
+		System.out.println("=========");
+		System.out.println("|SUCCESS|");
+		System.out.println("=========");
+		
+	}
+	
+	public void displayPrices(TicketPrice ticketPrice){
+		System.out.println("Students:\t\t" + ticketPrice.getStudentPricing());
+		System.out.println("Senior citizens:\t" + ticketPrice.getSeniorCitizenPricing());
+		System.out.println("Platinum ticket prices:\t" + ticketPrice.getPlatinumTicketPricing());
+		System.out.println("Thursday prices:\t" + ticketPrice.get_THU_pricing());
+		System.out.println("Sat and Sun prices:\t" + ticketPrice.get_SAT_SUN_PH_pricing());
+		System.out.println("Mon - Wed:\t\t" + ticketPrice.get_MON_WED_pricing());
+		System.out.println("Fri after 6:\t\t" + ticketPrice.get_FRI_EVE_PH_from_6_price());
+		System.out.println("Fri before 6:\t\t" + ticketPrice.get_FRI_EVE_PH_before_6_pricing());
+	}
+	
+	public void choicesForTicketPrices(){
+		System.out.println("1 - Students");
+		System.out.println("2 - Senior citizens");
+		System.out.println("3 - Platinum ticket pricing") ;
+		System.out.println("4 - Thursday prices");
+		System.out.println("5 - Sat and sun prices");
+		System.out.println("6 - Mon - wed");
+		System.out.println("7 - Fri after 6");
+		System.out.println("8 - Fri before 6");
+	}
+	
+	public void displayListOfPublicHolidays(){
+		SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.